packages/react-native-lockdown/README.md

```diff
@@ -0,0 +1,153 @@
+
+Upon inspecting the bundle, the reference error is throwing early when evaluating the SES shim.
+
+It is clear Babel is preparing it to be [transformed](https://github.com/endojs/endo/issues/662):
+
+```js
+// http://localhost:8081/index.bundle//&platform=android
+// ...
+(function (global) {
+  var _asyncToGenerator = require("@babel/runtime/helpers/asyncToGenerator");
+  var _toArray = require("@babel/runtime/helpers/toArray");
+  var _objectWithoutProperties = require("@babel/runtime/helpers/objectWithoutProperties");
+  var _toConsumableArray = require("@babel/runtime/helpers/toConsumableArray");
+  var _slicedToArray = require("@babel/runtime/helpers/slicedToArray");
+  var _defineProperty = require("@babel/runtime/helpers/defineProperty");
+  var _objectDestructuringEmpty = require("@babel/runtime/helpers/objectDestructuringEmpty");
+  var _excluded = ["name", "message", "errors", "cause", "stack"],
+    _excluded2 = ["random"],
+    _excluded3 = ["__options__"],
+    _excluded4 = ["errorTaming", "errorTrapping", "reporting", "unhandledRejectionTrapping", "regExpTaming", "localeTaming", "consoleTaming", "overrideTaming", "stackFiltering", "domainTaming", "evalTaming", "overrideDebug", "legacyRegeneratorRuntimeTaming", "__hardenTaming__", "dateTaming", "mathTaming"];
+  function _defineAccessor(e, r, n, t) { var c = { configurable: !0, enumerable: !0 }; return c[e] = t, Object.defineProperty(r, n, c); }
+  // ses@1.13.0
+  (function (functors) {
+// ...
+```
+
+Simply ensure Babel is configured to ignore your flavour of SES shim like in the [babel config][babel-config] example.
+
+Now it should look much cleaner:
+
+```js
+// http://localhost:8081/index.bundle//&platform=android
+// ...
+(function (global) {
+  // ses@1.13.0
+  (functors => options => {
+// ...
+```
+
+### Cannot read property 'setGlobalHandler' of undefined
+
+<img width="467" alt="Screenshot 2025-04-11 at 3 32 14 pm" src="https://github.com/user-attachments/assets/2bf45dd3-85b5-4e71-b76d-20961530df9f" />
+
+It looks like React Native is having trouble [initializing][init-core-ext] when setting up [error handling][set-up-error-handling-ext], due to the missing [error guard][rn-js-polyfills-error-guard-ext] polyfill.
+
+Did you remember to include the React Native JS polyfills in your custom config? See Hermes [example][hermes-example].
+
+[babel-config]: #babel-config
+[hermes-example]: #hermes
+[init-core-ext]: https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Core/InitializeCore.js
+[set-up-error-handling-ext]: https://github.com/facebook/react-native/blob/c50f3e5f668887bfb0c7080155c066a4fdcc092c/packages/react-native/Libraries/Core/setUpErrorHandling.js#L33
+[rn-js-polyfills-error-guard-ext]: https://github.com/facebook/react-native/blob/c50f3e5f668887bfb0c7080155c066a4fdcc092c/packages/polyfills/error-guard.js#L38-L40
```

Comment:
or
- better custom error (i.e. validatePolyfills)
- include peerDep @react-native/js-polyfills, if not included (trivial) and warn user